Mathlib/Data/Set/Image.lean

```diff
@@ -230,21 +233,17 @@ theorem _root_.Function.Commute.set_image {f g : α → α} (h : Function.Commut
     Function.Commute (image f) (image g) :=
   Function.Semiconj.set_image h
 
-/-- Image is monotone with respect to `⊆`. See `Set.monotone_image` for the statement in
-terms of `≤`. -/
-@[gcongr]
-theorem image_subset {a b : Set α} (f : α → β) (h : a ⊆ b) : f '' a ⊆ f '' b := by grind
-
-/-- `Set.image` is monotone. See `Set.image_subset` for the statement in terms of `⊆`. -/
-lemma monotone_image {f : α → β} : Monotone (image f) := fun _ _ => image_subset _
```

Comment:
Shouldn't this also be deprecated to `image_mono`? On the basis that even though it's not exactly `image_mono`, deprecating to `image_mono` is much more helpful than the function just randomly stopping working.